crapssim/bet.py

```diff
@@ -550,3 +550,59 @@ def _update_bet(self, table_object: "Table", dice_object: Dice) -> tuple[str | N
 
     def allowed(self, table: 'Table') -> bool:
         return table.new_shooter
+
+
+class AllBet(Bet):

```

Comment:
Maybe a bit pedantic, but what about changing this naming to `class ATS(Bet)`, and then changing the sub bets to `Tall(ATS)`, `Small(ATS)`, and `All(ATS)`?  I think this better reflects the naming conventions I see and leads to an easy to read strategy such as:

```
player.bet(All(player.unit))
player.bet(Tall(player.unit))
player.bet(Small(player.unit))
```

Reply:
Yeah, so I've never actually made this bet so I just copied what was printed on the felt in a google image search for it. I think if that you generally say All, Tall, and Small when making the bet that is consistent with the other bets being what is said when the bet is made (ex: Yo or Boxcars.) 

I'm a little concerned that users won't know the difference between All and ATS when importing and will think that ATS is meant to be All (although currently it's not much better.) Maybe the current AllBet should be something like ATSBase or something like that to differentiate it.

Reply:
Fair point about ATS as the base class.  I like ATSBase or ATSClass